addons-cluster/apecloud-mysql-cluster/templates/_helpers.tpl

```diff
@@ -62,10 +62,18 @@ raftGroup mode: max(replicas, 3)
 
 {{- define "apecloud-mysql-cluster.topology" }}
 {{- if and (eq .Values.mode "raftGroup") .Values.proxyEnabled }}
-  {{- if .Values.auditLogEnabled}}
-    {{- "apecloud-mysql-audit-with-proxy" }}
+  {{- if .Values.localEtcd }}
+    {{- if .Values.auditLogEnabled }}
+      {{- "apecloud-mysql-audit-proxy-etcd" }}
+    {{- else }}
+      {{- "apecloud-mysql-proxy-etcd" }}
+    {{- end }}
   {{- else }}
-    {{- "apecloud-mysql-with-proxy" }}
+    {{- if .Values.auditLogEnabled }}
+      {{- "apecloud-mysql-audit-with-proxy" }}
+    {{- else }}
+      {{- "apecloud-mysql-with-proxy" }}
+    {{- end }}
```

Comment:
It seems that the composition of options has resulted in an excessive number of cmpd